### ISSUE TITLE
Parenthesized types are not indented

### ISSUE THREAD

**Issue by AdamNiederer** [orzechowskid/tsi.el#25]
`tsx-mode` does not indent empty or non-empty lines within a parenthesized type:
```ts
let a: (
0
)
```
`typescript-mode` indents it like this:
```ts
let a: (
  0
)
```
`tsx-mode` does indent all of these cases correctly:
```ts
let a: [
  0
]

let a: {
  a: 0
}

let a = (
  0
)
```

**Comment by stevemolitor** [comment]
Thanks for all the good bug reports @AdamNiederer.

Fix for this one here: #26.

**Comment by AdamNiederer** [comment]
Thanks for fixing them!
